bentoml/_internal/frameworks/catboost.py

```diff
@@ -227,59 +215,43 @@ def save(
     return _model.tag
 
 
-class _CatBoostRunner(Runner):
+class _CatBoostRunner(BaseModelRunner):
     @inject
     def __init__(
         self,
-        tag: Tag,
+        tag: t.Union[str, Tag],
```

Comment:
Should we make everything below kwarg only?

```suggestion
        tag: t.Union[str, Tag],
        *,
```